flink-dist/src/main/resources/META-INF/NOTICE

```diff
@@ -41,7 +41,7 @@ The following dependencies all share the same BSD license which you find under l
 - org.scala-lang:scala-library:2.11.12
 - org.scala-lang:scala-reflect:2.11.12
 - org.scala-lang.modules:scala-java8-compat_2.11:0.7.0
-- org.scala-lang.modules:scala-parser-combinators_2.11:1.0.4
+- org.scala-lang.modules:scala-parser-combinators_2.11:1.1.1
```

Comment:
Upgrading from `1.0.4` to `1.1.1` is not a problem?

Reply:
Not during runtime since akka is the only runtime dependency requiring this. The other dependency that needs this is `org.scalatest:scalatest_2.11` but that's a test dependency, and since the tests are passing we should be good.

Reply:
`scala-compiler´ also needs it, but that should be covered by tests.

Reply:
The table API is not relying on Scala's parser combinators?

Reply:
right, they do, but without declaring the dependency 👿 , but given how it's in a rather central place I'd again assume that the tests cover this sufficiently.